lib/watcher.ex

```diff
@@ -4,7 +4,7 @@ defmodule Watcher do
   def callback(file, events)  do
     if Enum.member?(events, :modified) do
       try do
-        [{mod, _}] = Code.load_file(file)
+        [{mod, _} | _] = Code.load_file(file)
```

Comment:
Structs are defined within a module, when I add a 
```
defmodule MyStruct do
  defstruct ...
defmodule
``` 
within a koan I get more than a single module. 
The previous version forced a single element in the list and failed the match.